wai-aria/scripts/aria-utils.js

```diff
@@ -51,6 +51,7 @@ const AriaUtils = {
       }
       promise_test(async t => {
         const expectedRole = el.getAttribute("data-expectedrole");
+        assert_not_equals(expectedRole, null, `Element should have attribute \'${attributeName}\`. Element: ${el.innerHTML}`);
```

Comment:
Rather than fail the test with an `assert`, I think we should throw after line 47 above. You can see on line 44, that we throw when someone is used in correct mark up or used the API verifyRolesBySelector incorrectly -- a failing test should indicate the browser did something wrong, not the test author :)

Reply:
The convenience methods in /wai-aria/scripts/aria-utils.js already handle this.

Look at some of the other ARIA or AccName tests instead of adding your own asserts. E.g.
```js
AriaUtils.verifyRolesBySelector(".ex")
```

Reply:
Ha... Sorry for my dufus reply... I was clearly not paying close enough attention.

Reply:
Ha... Sorry for my prior absent-minded and irrelevant reply... I was clearly not paying close enough attention.